src/game/Editor/SmartMethod.cc

```diff
@@ -34,8 +34,8 @@ void CalcSmartBrokenWallDefault( UINT16 *pusObjIndex, UINT16 *pusUseIndex )
 	switch( gubBrokenWallUIValue )
 	{
 		case 0:case 1: *pusUseIndex = 49 + gubBrokenWallUIValue;	break;
-		case 3:				 *pusUseIndex = 62;													break;
-		case 4:				 *pusUseIndex = 64;													break;
+		case 3:		*pusUseIndex = 62;				break;
+		case 4:		*pusUseIndex = 64;				break;
```

Comment:
I've discovered several anomalies of that type. We should definitely make a list. Can you start it?